### ISSUE TITLE
[Issue] release 2.4.6 not stable

### ISSUE THREAD

**Issue by chenrui333** [projectdiscovery/subfinder#396]
it looks like release 2.4.6 tag is not quite stable.

relates to https://github.com/Homebrew/homebrew-core/pull/71628

**Comment by ehsandeep** [comment]
Hi @chenrui333,

We pushed few changes after release and recreated the release again, and that might caused an issue? as of now things looks stable.

**Comment by choopm** [comment]
@ehsandeep 
Maybe that's the reason for this? Running on go 1.16
```
go get -u github.com/projectdiscovery/subfinder/v2/cmd/subfinder@v2.4.6
go: downloading github.com/projectdiscovery/subfinder/v2 v2.4.6
go get github.com/projectdiscovery/subfinder/v2/cmd/subfinder@v2.4.6: github.com/projectdiscovery/subfinder/v2@v2.4.6: verifying module: checksum mismatch
	downloaded: h1:cbD0WBZqt2gm0soHcsLg5KDyBeTOMIPkUNlAw3kkL6Q=
	sum.golang.org: h1:FMi4v8fT5aq3QSjvAWZH0GMIAGoQRJxLtWWIRLv8ejo=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.
```

**Comment by ehsandeep** [comment]
We fixed the installation issue with  [v2.4.7](https://github.com/projectdiscovery/subfinder/releases/tag/v2.4.7) release.

@chenrui333 @choopm FYI^

**Comment by alizaiyousaf35-hash** [comment]
Go install
